### DISCUSSION TITLE
Installation of KnowWE on a windows server

### DISCUSSION THREAD

**Discussion by VitaliyMez** [denkbares/d3web-KnowWE#10] [Q&A] [answered]
Hi,

I want to install KnowWE on the windows IIS server, without TomCat.
Is that possible?
If not, any advice on free TomCat hosting?

Thank you!

**Comment by AlbrechtStriffler** [comment, marked as answer]
Hi Vitaliy,
KnowWE is a Java EE Web App coming as a WAR file. To my knowledge, windows IIS cannot host WAR files. Also see this post, where someone had a similar question: https://social.msdn.microsoft.com/Forums/en-US/368c4627-6b67-4609-87ff-b16820e659b5/how-to-deploy-war-file-on-iis-8?forum=iis7general

I don't know of any free Tomcat hosting, but Tomcat itself is free. So everywhere you can run IIS, you should also be able to run Tomcat. Tomcat is even provided on the downloadable distribution package...

Regards
Albrecht

**Reply by VitaliyMez** [reply]
Thank you, Albrecht!
